src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -596,3 +606,16 @@ file to obtain scanning parameters. \_epi files can be a 3D or 4D - in the
 latter case all timepoints share the same scanning parameters. To indicate which
 run is intended to be used with which functional or diffusion scan the
 IntendedFor field in the JSON file should be used.
+
+<!-- Link Definitions -->
+
+[deprecated]: https://en.wikipedia.org/wiki/Deprecation
```

Comment:
For a separate PR: Might be good to have a note on deprecation in the spec itself. Something like:

> DEPRECATED - A "deprecated" entity or metadata field SHOULD NOT be used in the generation of new datasets. It remains in the standard in order to preserve the interpretability of existing datasets. Validating software SHOULD warn when deprecated practices are detected and provide a suggestion for updating the dataset to preserve the curator's intent.